pkg/openapi/server.spec.yaml

```diff
@@ -2339,6 +2306,7 @@ components:
       type: object
       required:
       - networkIds
+      - id
```

Comment:
`id` doesn't seem to be amongst the properties in this object. This might be a remnant from getting code generation to do what you wanted.